website/src/docs/hotchocolate/v14/migrating/migrate-from-13-to-14.md

```diff
@@ -144,17 +171,27 @@ We have aligned all builder APIs to be more consistent and easier to use. Builde
 
 The interface `IQueryRequestBuilder` and its implementations were replaced with `OperationRequestBuilder` which now supports building standard GraphQL operation requests as well as variable batch requests.
 
-The `Build()` method returns now a `IOperationRequest` which is implemented by `OperationRequest` and `VariableBatchRequest`.
+The `Build()` method now returns a `IOperationRequest` which is implemented by `OperationRequest` and `VariableBatchRequest`.
+
+We've also renamed and consolidated some methods on the `OperationRequestBuilder`:
 
-We have also simplified what the builder does and removed a lot of the convenience methods that allowed to add single variables to it. This has todo with the support of variable batching. Now, you have to provide the variable map directly.
+| Before                              | After                                                                       |
+| ----------------------------------- | --------------------------------------------------------------------------- |
+| `SetQuery("{ __typename }")`        | `SetDocument("{ __typename }")`                                             |
+| `AddVariableValue("name", "value")` | `AddVariableValues(new Dictionary<string, object?> { ["name"] = "value" })` |
 
 ### IQueryResultBuilder replaced by OperationResultBuilder
 
 The interface `IQueryResultBuilder` and its implementations were replaced with `OperationResultBuilder` which produces an `OperationResult` on `Build()`.
 
 ### IQueryResult replaced by OperationResult
 
-The interface `IQueryResultBuilder` and its implementations were replaced with `OperationResultBuilder` which produces an `OperationResult` on `Build()`.
+The interface `IQueryResult` has been replaced by `OperationResult`.
```

Comment:
Would it be more accurate to say it was replaced with `IOperationResult` and its implementation `OperationResult`.

Tests do mock various things here.

Reply:
Already done by Glen in another PR